### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
The build breaks with: 

> TypeError: IntlDateFormatter::__construct() expects parameter 6 to be string, null given

I don't understand why the build would pass on 3.x

**Comment by greg0ire** [comment]
Ah I understand, it's about the "strict types" declaration.

File: README.md [hunk 1]
```diff
@@ -39,3 +39,8 @@ This package is available under the [MIT license](LICENSE).
 [coveralls_stable_link]: https://coveralls.io/github/sonata-project/SonataCoreBundle?branch=3.x
 [coveralls_unstable_badge]: https://coveralls.io/repos/github/sonata-project/SonataCoreBundle/badge.svg?branch=master
 [coveralls_unstable_link]: https://coveralls.io/github/sonata-project/SonataCoreBundle?branch=master
+
+[scrutinizer_stable_badge]: https://scrutinizer-ci.com/g/sonata-project/SonataCoreBundle/badges/quality-score.png?b=3.x
```
Comment (OskarStark) [inline README.md]: this looks wrong

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-Sphinx
+Sphinx!=1.8.0
 git+https://github.com/fabpot/sphinx-php.git
-sphinx_rtd_theme
\ No newline at end of file
+sphinx_rtd_theme
```
Comment (OskarStark) [inline docs/requirements.txt]: the dev kit changes should be available as they get executed against `master` separately

Comment (greg0ire) [reply docs/requirements.txt]: What should I do? Won't dev-kit fix anything that goes wrong?

File: README.md (hunk 1 above)
Comment (greg0ire) [reply README.md]: Should it be master?

File: docs/requirements.txt (hunk 2 above)
Comment (OskarStark) [reply docs/requirements.txt]: Ofc it will, but only when it gets executed again

File: README.md (hunk 1 above)
Comment (OskarStark) [reply README.md]: yes and it is currently available, maybe your branch was outdated? 🤔

Comment (OskarStark) [reply README.md]: ah sorry, no it isn't available in `master`, but yes it should be `master` in the URL

Comment (greg0ire) [reply README.md]: But it says "stable", and master is unstable.

Comment (OskarStark) [reply README.md]: argh you are right, sorry for confusing.

Everything is fine 👍

**Pull request merged**